Userland/Libraries/LibSoftGPU/Light/Light.h

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2022, Jesse Buhagiar <jooster669@gmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <LibGfx/Vector4.h>
+
+namespace SoftGPU {
+
+struct Light {
+    bool is_on { false };
+
+    // According to the OpenGL specification, page 56, all of the parameters
```

Comment:
Maybe mention the `1.5` version of the spec here, since there are a lot of OpenGL specs to be found